framework/platform/bpl/uci/scripts/platform_daemon_ugw.sh

```diff
@@ -0,0 +1,146 @@
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"operating_mode" : "Gateway"}}'
+    ubus call uci commit '{"config" : "beerocks" }'
+    
+    #configure wireless
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"disabled" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"disabled" : "1"}}'
+    
+    #uci commit wireless (currently automatically restarts the hostapd's)
+    ubus call uci commit '{"config" : "wireless" }'
+    # If the automatic restard is removed, the follwoing should be used after commit
+    #/etc/init.d/network reload
+
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Set bridge ip to 192.168.1.1
+    #echo "Set bridge ip to 192.168.1.1"
+    #ifconfig br-lan 192.168.1.1
+    
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # start DHCP server
+    #/etc/init.d/odhcpd start
+    
+    # Start APM
+    echo "Start APM"
+    /opt/prplMesh/scripts/prplmesh_utils.sh start
+}
+static_init()
+{
+    # Check if repeater mode (and required functionality is supported)
+    if [ $(uci get beerocks.config.management_mode) != "Proprietary-Mesh" ]
+    then 
+        echo "not Repeater"
+        exit 1;
+    fi
+    echo "I am repeater!"
+    # Kill APM
+    echo "Kill APM"
+    /opt/prplMesh/scripts/prplmesh_utils.sh stop
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Kill DHCP server
+    #echo "Kill DHCP server"
+    #/etc/init.d/odhcpd stop 
+}
+start()
+{
+    # Check if repeater mode (and required functionality is supported)
+    if [ $(uci get beerocks.config.management_mode) != "Proprietary-Mesh" ]
+    then 
+        echo "not Repeater"
+        exit 1;
+    fi
+    if [ $(uci get beerocks.config.operating_mode) != "WDS-Repeater" ]
+    then 
+        echo "not Repeater"
+        exit 1;
+    fi
+    if [ -z $(which wpa_supplicant) ]
+    then 
+        echo "no wpa_supplicant"
+        exit 1;
+    fi
+    if [ -z $(uci show wireless | grep sta) ]
+    then 
+        echo "station not configurated, try to run /opt/intel/wave/scripts/wave_factory_reset.sh"
+        exit 1;
+    fi
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    #add interfaces to bridge
+    #brctl addif br-lan wlan1
+    #brctl addif br-lan wlan3
+    
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Start dhcp client
+    #echo "Start dhcp client"
+    #udhcpc -p /var/run/udhcpc-br-lan.pid -i br-lan &
+    
+    # Start APM
+    echo "Start APM"
+    /opt/prplMesh/scripts/prplmesh_utils.sh start
+}
+case $1 in
+        "ap_to_repeater")
+                ap_to_repeater
+                ;;
+        "repeater_to_ap")
+                repeater_to_ap
+                ;;
+        "start")
+                start
+                ;;
```

Comment:
If there's a start, there should also be a stop.

Reply:
> If there's a start, there should also be a stop.

I have removed the start as the script should only configure the platform to repeater mode and not run the prplmesh which is a separate operation of the system/user